libs/generators/src/lib/generators/generators-menu/generators-menu.component.scss

```diff
@@ -0,0 +1,14 @@
+:host {
+    .container {
+        display: flex;
+        flex-direction: column;
+        gap: 8px;
+        height: 100%;
+    }
+
+    .menu-item {
+        width: 100%;
+        justify-content: left;
+        width: 10rem;
```

Comment:
You have width twice, and anyway please try to use `rem` but prefer `%` especially when it's flex

Reply:
@iYadgar thanks, small mistake